From fec18856e033dc7a487670e563218f5b9026c0fd Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Thu, 30 Aug 2018 14:27:02 -0500
Subject: [PATCH] overlayroot: fstab: use fs-virtual for debugfs entries

On some embedded systems, it's common to mount /sys/kernel/debug/ as debugfs,
in overlayroot lets not rewrite this as an overlayfs...

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 overlayroot/scripts/init-bottom/overlayroot | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/overlayroot/scripts/init-bottom/overlayroot b/overlayroot/scripts/init-bottom/overlayroot
index 0628238..c13c02b 100755
--- a/overlayroot/scripts/init-bottom/overlayroot
+++ b/overlayroot/scripts/init-bottom/overlayroot
@@ -331,7 +331,7 @@ overlayrootify_fstab() {
 		else
 			case "$vfstype" in
 				vfat|fat) use_orig="fs-unsupported";;
-				proc|sys|tmpfs|dev|udev) use_orig="fs-virtual";;
+				proc|sys|tmpfs|dev|udev|debugfs) use_orig="fs-virtual";;
 			esac
 		fi
 
-- 
2.18.0

